english/talk_to_neighbour/talk_to_neighbour.md

```diff
@@ -1,3 +1,9 @@
 I hear you
 I see you
-I'm Listening
\ No newline at end of file
+I'm Listening
+
+As you speak to your neighbor, you see a dark car approach. Two figures emerge, a man and a woman.
+
+[Talk to the man](english/talk_to_neighbor/woman/scully.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.